db-manager/tasks/dailyCheckInTask.js

```diff
@@ -149,13 +155,22 @@ async function dailyCheckInTask(taskInput, db) {
           let totalDailyXp = 0;
 
           const allXCallAddresses = [];
-          validUser.linkedWallets.forEach(xChainWallet => {
-            if (xChainWallet.type === "evm") {
-              allXCallAddresses.push(...chains.evm.map(chain => `${chain}/${xChainWallet.address}`));
-            } else if (["sui", "stellar"].includes(xChainWallet.type)) {
-              allXCallAddresses.push(`${xChainWallet.type}/${xChainWallet.address}`)
-            }
-          })
+
+          if (SEED_ID === "DAILY_CHECK_IN_SUI_COLLATERAL") {
```

Comment:
instead using string literals, we can import the value from `common/utils/config.js` on the `tasks` object